packages/@lwc/engine-server/src/__tests__/fixtures/render-correct-tags/modules/x/elements/elements.html

```diff
@@ -0,0 +1,22 @@
+<template>
+    <!-- Empty non-void HTML namespace element -->
+    <a href="https://www.salesforce.com/"></a>
+    <!-- Non-empty non-void HTML namespace element -->
+    <label for="textInput">Some input</label>
+    <!-- Void HTML namespace element -->
+    <input id="textInput" type="text">
+    <!-- Foreign namespace elements -->
+    <svg>
+        <!-- Non-empty foreign element -->
+        <pattern>
+            <image xlink:href="https://www.salesforce.com/"></image>
+        </pattern>
+        <!-- Empty foreign element -->
+        <image xlink:title="title"></image>
+    </svg>
+    <!-- Rendered with inner-html -->
+    <svg>
+        <pattern lwc:inner-html={foreignNamespaceInnerHtml}></pattern>
+        <image xlink:title="title"></image>
+    </svg>
```

Comment:
It'd be great to add a MathML block here for completeness.

Reply:
including `<math>` namespace elements passes for `ssr-compiler` but generates a warning when `engine-server` fixtures are ran as it appears MathML elements [is not recognized as a knownTag](https://github.com/salesforce/lwc/blob/77010690be2e23e42f1d03fd83f3a7dcd098428a/packages/%40lwc/template-compiler/src/parser/index.ts#L1599) by template-compiler -> validateElement. To enable testing this I suppressed the warning [here](https://github.com/salesforce/lwc/pull/4992/files#diff-4c1b19c26186adbadb3762e71fd41a26cf8ab00fc7ee5a790ef1afcf14c606a0R85). 

Is that OK or would you rather modify template-compiler -> validateElement to recognize MathML elements, or...?

Reply:
Yep, opened an issue for this: https://github.com/salesforce/lwc/issues/5010

MathML should be supported.

Reply:
Ok thanks, seems wrong. I updated the warning suppression in the fixture to reference the issue you opened.